### PR TITLE
Added reolink.client version 8.3.1

### PR DESCRIPTION
**Pull request by andrea-capigiri**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25705)

### CODE REVIEW

File: manifests/r/Reolink/Client/8.3.1/Reolink.Client.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+PackageIdentifier: Reolink.Client
+PackageVersion: 8.3.1
+PackageLocale: en-US
+Publisher: Reolink
+PublisherUrl: https://reolink.com/
+PublisherSupportUrl: https://support.reolink.com/hc/en-us/
+PrivacyUrl: https://reolink.com/privacy-policy/
+#Author: 
+PackageName: Reolink Client
```
Comment (ImJoakim) [inline manifests/r/Reolink/Client/8.3.1/Reolink.Client.locale.en-US.yaml:12]: image

```suggestion
PackageName: Reolink
```

File: manifests/r/Reolink/Client/8.3.1/Reolink.Client.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created using YamlCreate.ps1 v1.1.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Reolink.Client
+PackageVersion: 8.3.1
+PackageLocale: en-US
+Publisher: Reolink
```
Comment (ImJoakim) [inline manifests/r/Reolink/Client/8.3.1/Reolink.Client.locale.en-US.yaml:7]: image

```suggestion
Publisher: Shenzhen Reolink Technology Co., Ltd.
```

**Review by ImJoakim — CHANGES_REQUESTED** [review]
It seems like you pushed `Mozilla.Thunderbird` to the same PR as `Reolink.Client`, please remove `Mozilla.Thunderbird` and make a new branch/pr containing `Mozilla.Thunderbird`.

**Review by ImJoakim — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/25705#pullrequestreview-735979845

**Review by andrea-capigiri — COMMENTED** [review]
Fixed Publisher and PackageName for Reolink version 8.3.1

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
`The pull request contains more than one manifest. Either singleton or multi file manifest under one version directory.`

https://github.com/microsoft/winget-pkgs/pull/25705#pullrequestreview-735979845

**Pull request closed without merging**